tools/osbuilder/rootfs-builder/nvidia/nvidia_rootfs.sh

```diff
@@ -151,14 +154,8 @@ setup_nvidia_gpu_rootfs_stage_one() {
 	mount --make-rslave ./dev
 	mount -t proc /proc ./proc
 
-	local driver_version="latest"
-	if echo "$NVIDIA_GPU_STACK" | grep -q '\<latest\>'; then
-		driver_version="latest"
-	elif echo "$NVIDIA_GPU_STACK" | grep -q '\<lts\>'; then
-		driver_version="lts"
-	fi
-
-	chroot . /bin/bash -c "/nvidia_chroot.sh $(uname -r) ${run_file_name} ${run_fm_file_name} ${ARCH} ${driver_version}"
+	chroot . /bin/bash -c "/nvidia_chroot.sh $(uname -r) ${run_file_name} \
```

Comment:
It looks like `ARCH` isn't set in this script. I guess it should inherit it from rootfs.sh envs, but I wonder if it's worth adding `arch=${ARCH:-$(uname -m)}` at the top to double check it isn't unset as we don't have `set -u`/`set -o nounset` in the script?

Reply:
Good point. Adding `set -u`